src/soko_game.rs

```diff
@@ -117,7 +124,7 @@ pub fn handle_event(model: &mut Model) -> io::Result<Option<GameAction>> {
     }
 
     // Prevent handling key events, coincidentally, because it's solved!
-    if window.world.is_sokoban_solved() {
+    if soko_game.current_state.is_sokoban_solved() {
```

Comment:
im proud of your abstractions. compare soko_game.current_state.is_solved to 
window.world.is_solved. you evaporated the uncertainty of the names. 

its so clear that we are checking the game of sokoban, to see its current world (which we rightfully call its state) and see if its solved.